lib/orbit_web/controllers/react_app_controller.ex

```diff
@@ -4,6 +4,10 @@ defmodule OrbitWeb.ReactAppController do
   def home(conn, _params) do
     # The home page is often custom made,
     # so skip the default app layout.
-    render(conn, :react_app, layout: false)
+    render(conn, :react_app,
+      layout: false,
+      sentry_dsn: Application.get_env(:sentry, :dsn),
+      sentry_environment: Application.get_env(:sentry, :environment_name)
```

Comment:
In Glides, we end up referring to `sentry_release` in a bunch of non-sentry places, because sentry was the first use of a release name. But it's not really a sentry-specific concept, so the name is a bit misleading.

I'd like to try to fix that problem in orbit. The environment is pretty generic concept. Can we add a `config :orbit, environment: System.get_env("ENVIRONMENT")` in `runtime.exs` so that we can refer to this everywhere as `environment: Application.get_env(:orbit, :environment)` and not tie it to the sentry name?

Reply:
yep! fixed in above.